runtime/trampolines-varargs.h

```diff
@@ -16,7 +16,7 @@ struct XamarinCallState {
 	union {  // floating point return value
 		double double_ret;
 		float float_ret;
-		long long longlong_ret;
+		uint64_t ulonglong_ret;
```

Comment:
What were the sign conversions warnings? Seems that fixing those without changing the field type would be a  easier and shorter fix.

Reply:
See https://github.com/xamarin/xamarin-macios/pull/7428#pullrequestreview-316590686